asu/api.py

```diff
@@ -74,6 +74,48 @@ def get_queue() -> Queue:
     return g.queue
 
 
+def validate_packages(req):
+    if "packages" in req and "packages_versions" in req:
+        return (
+            {
+                "status": "duplicate_package_list",
+                "message": "Use either `packages` or `packages_versions`, not both",
+            },
+            400,
+        )
+    elif "packages" in req:
+        pass
+    elif "packages_versions" in req:
+        req["packages"] = req["packages_versions"].keys()
+    else:
+        # no packages to check
+        return
+
+    req["packages"] = set(req["packages"]) - {"kernel", "libc", "libgcc"}
```

Comment:
Shouldn't this contain `libpthread` as well? I filter those already in `auc`.

Reply:
I never got a error here but sure can add that too